### PR TITLE
New pull request with Hyea-Hyun's improvements plus a few of my own v2

### PR DESCRIPTION
**Pull request by rymuelle**
This pull request contains the improvements in Hyea-Hyun's last pr plus a few of my own additions to run the code. I can confirm it recreates Hyea-Hyun's cutflow numbers. 

I have not had time to fully go through the issues raised previously and the code needs cleaning up a bit.

Description
I added in a "makeHists.py" code and a python dict file to manage xsections and run the code automatically.

Types of changes
  Bug fix (non-breaking change which fixes an issue)
  New feature (non-breaking change which adds functionality)
  Breaking change (fix or feature that would cause existing functionality to change)
Motivation and Context
Get dev up to speed with latest code dealing with xsections.

How Has This Been Tested?
It produces the histograms with the correct scaling.

Screenshots (if appropriate):
Checklist:
  My code follows the code style of this project.
[x ] My change requires a change to the documentation.
  I have updated the documentation accordingly.
  I have added tests to cover my changes.
  All new and existing tests passed.

### CODE REVIEW

File: MakeHist/makeHists.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  if not exists("output"):
+    mkdir("output")
+  instance.Loop("output/{0}_{1}".format(cutString,l_sampleDict['outName']),l_sampleDict['xsection'],lumi,l_sampleDict['totalEvents'], cutString)#
+
+
+cutStrings = ["b=1_j=0",
+"b=1,2_b+j=2"]
+
+cutStrings = ["1",
```
Comment (rathjd) [inline MakeHist/makeHists.py]: Why are there two lists of strings that are set to overwrite one another?

File: MakeHist/makeHists.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      chain.Add(pathString)
+
+  instance = mumu(chain)
+
+  if not exists("output"):
+    mkdir("output")
+  instance.Loop("output/{0}_{1}".format(cutString,l_sampleDict['outName']),l_sampleDict['xsection'],lumi,l_sampleDict['totalEvents'], cutString)#
+
+
+cutStrings = ["b=1_j=0",
```
Comment (rathjd) [inline MakeHist/makeHists.py]: Just remove this, because you numbered it down later.

File: MakeHist/makeHists.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+"b=1,2_b+j=2"]
+
+cutStrings = ["1",
+"2",
+"3",
+"4",
+"5",
+""]
+
```
Comment (rathjd) [inline MakeHist/makeHists.py:40]: Why do you have two lists here that don't append, but rather overwrite one another? Unless I misunderstand python here.

**Review by mexanick — APPROVED** [review]
I approve this PR and we move to discussed issue-PR scheme in future

**Pull request merged**